rs/execution_environment/BUILD.bazel

```diff
@@ -101,7 +101,7 @@ ENV = {}
 
 rust_library(
     name = "execution_environment",
-    srcs = glob(["src/**"]),
+    srcs = glob(["src/**/*.rs"]),
```

Comment:
Is this a drive-by improvement? I suppose this way we don't include some test files in the lib target, right? If yes, I would generally recommend to try to do these in separate PRs as a rule of thumb.

Reply:
>  I suppose this way we don't include some test files in the lib target, right?

We include all files currently checked-in. The motivation for this is that vim adds .swp file for every file currently edited and those file confuse bazel.

Reply:
That seems like a potentially general issue then that might exist in other `rust_library` targets. It's also somewhat specific to what editor people use. Perhaps something to tackle separately really then, potentially in a big PR for all such relevant targets?

Reply:
I've reverted the change.